k8s/helmfile/argo-cd.yaml

```diff
@@ -0,0 +1,29 @@
+environments:
+  default:
+    kubeContext: INVALID-ENVIRONMENT
+  production:
+    kubeContext: gke_wikibase-cloud_europe-west3-a_wbaas-3
+    values:
+      - ./env/production/base.yaml
+      - ./env/production/private.yaml
+  staging:
+    kubeContext: gke_wikibase-cloud_europe-west3-a_wbaas-2
+    values:
+      - ./env/staging/base.yaml
+      - ./env/staging/private.yaml
+  local:
+    kubeContext: minikube-wbaas
+    values:
+      - ./env/local/base.yaml
+      - ./env/local/private.yaml
+
+---
+repositories:
```

Comment:
Just out of curiosity: is there a rule for when to declare a repository in the sub-Helmfile and when to do it in the top level one? Right now, we sometimes do the one and sometimes the other.

Reply:
I think we're wildly inconsistent but...
on a totally clean install because the sub-helmfiles are evaluated first it's necessary to have any repositories mentioned in the sub-helmfile or helm will error when evaluating it.

However you can get away with mentioning it in the top-level one because the repository definitions persist between runs of helm so the *second time* you can put it in either place.

Really it should be in both because sub-helmfiles should be standalone